Init tests

GeorgiyMalinovskiy opened the pull request:
new file:   .gitignore
new file:   __tests__/constructor.test.py
new file:   __tests__/login.test.py
new file:   __tests__/profile.test.py
new file:   __tests__/register.test.py
new file:   conftest.py
new file:   locators.py

Commit:
Init tests:
new file:   .gitignore
new file:   __tests__/constructor.test.py
new file:   __tests__/login.test.py
new file:   __tests__/profile.test.py
new file:   __tests__/register.test.py
new file:   conftest.py
new file:   locators.py

k-asadova commented on __tests__/constructor.test.py:
```diff
@@ -0,0 +1,38 @@
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
```
Нужно поправить:

1) директория называется tests, без подчеркиваний

2) название файла должно начинаться с test_ - чтобы pytest автоматически их собирал как тестовые

GeorgiyMalinovskiy replied:
Исправлено

k-asadova commented on __tests__/login.test.py:
```diff
@@ -0,0 +1,75 @@
+        submit_button.click()
+
+        # Проверка успешного входа
+        WebDriverWait(driver, 5).until(expected_conditions.presence_of_element_located(MainPageLocators.CONSTRUCTOR_BUTTON))
```
Нужно поправить: в конце каждого теста - assert, добавь прямо перед WebDriverWait это ключевое слово, и в других тестах аналогично

GeorgiyMalinovskiy replied:
Дополнил

Commit:
move/update:

__tests__/constructor.test.py -> tests/test_constructor.py
__tests__/login.test.py -> tests/test_login.py
__tests__/profile.test.py -> tests/test_profile.py
__tests__/register.test.py -> tests/test_register.py
